test/integration/DCAHubSwapper/multi-pair-swap-with-dex.spec.ts

```diff
@@ -58,6 +58,7 @@ contract('Multi pair swap with DEX', () => {
     //We are setting a very high fee, so that there is a surplus in both reward and toProvide tokens
     await DCAHub.connect(timelock).setSwapFee(50000); // 5%
     // Allow swapper
+    await DCAHub.connect(governor).grantRole(await DCAHub.PRIVILEGED_SWAPPER_ROLE(), DCAHubSwapper.address);
```

Comment:
If I didn't add this, the test would be flaky. Depending on how fast the test was executed, it would fail 😅